src/trials.jl

```diff
@@ -262,10 +285,10 @@ function prettymemory(b)
     return string(@sprintf("%.2f", value), " ", units)
 end
 
-Base.show(io::IO, t::Trial) = print(io, "Trial(", prettytime(time(t)), ")")
-Base.show(io::IO, t::TrialEstimate) = print(io, "TrialEstimate(", prettytime(time(t)), ")")
-Base.show(io::IO, t::TrialRatio) = print(io, "TrialRatio(", prettypercent(time(t)), ")")
-Base.show(io::IO, t::TrialJudgement) = print(io, "TrialJudgement(", prettydiff(time(ratio(t))), " => ", time(t), ")")
+Base.show(io::IO, t::Trial) = print(io, "Trial(", prettytime(realtime(t)), ")")
```

Comment:
Should these `show` methods include `cputime`?

Reply:
maybe `cpuratio`

Reply:
On playing with it I think it is not necessary.